docs/src/differentiable_programming.md

```diff
@@ -201,7 +201,7 @@ julia> function energy_at_final_time(k) # k is the wave number of the initial co
            mesh = TreeMesh((-1.0, -1.0), (1.0, 1.0), initial_refinement_level=3, n_cells_max=10^4)
            solver = DGSEM(3, flux_lax_friedrichs)
            initial_condition = (x, t, equation) -> begin
-               x_trans = Trixi.x_trans_periodic_2d(x - equation.advectionvelocity * t)
+               x_trans = Trixi.x_trans_periodic_2d(x - equation.advection_velocity * t)
```

Comment:
Needs a NEWS.md entry since it's changing documented stuff (documented here)

Reply:
Where exactly should I add it there?

Reply:
Under `Changed` in `Changes when updating to v0.4 from v0.3.x`